src/node/chainstatemanager_args.cpp

```diff
@@ -32,10 +32,11 @@ util::Result<void> ApplyArgsManOptions(const ArgsManager& args, ChainstateManage
     if (auto value{args.GetBoolArg("-checkpoints")}) opts.checkpoints_enabled = *value;
 
     if (auto value{args.GetArg("-minimumchainwork")}) {
-        if (!TrySanitizeHexNumber(*value)) {
+        if (auto sanitized_hex{TrySanitizeHexNumber(*value, /*result_size=*/64)}) {
```

Comment:
in the next commit we're using `uint256::size() * 2` instead of the hard-coded 64 - consider unifying, if you agree that it documents the behavior slightly better